src/tools/pub/cmd/pub-downgrade.md

```diff
@@ -6,7 +6,7 @@ description: Use pub downgrade to get the lowest versions of all dependencies us
 _Downgrade_ is one of the commands of the [pub tool](/tools/pub/cmd).
 
 {% prettify sh %}
-$ pub downgrade [args] [dependencies]
+$ pub downgrade [args] [dependencies] [--offline] [--no-offline] [--dry-run]
```

Comment:
Hi. Thanks for the PR.

As I mentioned in https://github.com/dart-lang/site-www/pull/2099#issuecomment-554756767, I'd suggest using the same syntax as the command help, namely `--[no-]offline`, etc.

- The `args` were meant to represent the options. If you're explicitly listing the options, they need to appear before the dependencies (and so drop `args`)
- dependencies -> dependencies... 
- Maybe include the shorthand too: `--dry-run` -> `-n|--dry-run`? @kwalrath WDYT?

---

```console
> pub downgrade -h
Downgrade the current package's dependencies to oldest versions.

This doesn't modify the lockfile, so it can be reset with "pub get".

Usage: pub downgrade [dependencies...]
-h, --help            Print this usage information.
    --[no-]offline    Use cached packages instead of accessing the network.
-n, --dry-run         Report what dependencies would change but don't change any.

Run "pub help" to see global options.
See https://dart.dev/tools/pub/cmd/pub-downgrade for detailed documentation.
```

Reply:
Hi @chalin,

Thanks for your guidance! Pending @kwalrath 's input, I will amend the line to read:

`
$ pub downgrade [--[no-]offline] [-n|--dry-run] [dependencies...] 
`

Reply:
@jtxmp I'll be happy to review again once you've made the changes @chalin suggested.